### PR TITLE
V2.11.1

### PR DESCRIPTION
**Pull request by Jakohbae**
Allow for propagating timeout for ssh connection from GerritQueryHandler down to jsch.Session.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -84,7 +84,6 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-jdk14</artifactId>
             <version>1.6.1</version>
-            <scope>test</scope>
```
Comment (rsandell) [inline pom.xml]: Why?!
This dependency should only be in the top app layer whenever someone decides what bridge to use.

File: src/main/java/com/sonymobile/tools/gerrit/gerritevents/ssh/SshConnectionFactory.java [hunk 2]
```diff
@@ -74,7 +74,26 @@ public static SshConnection getConnection(String host, int port,
      */
     public static SshConnection getConnection(String host, int port, String proxy,
                                               Authentication authentication) throws IOException {
-        return getConnection(host, port, proxy, authentication, null);
+        return getConnection(host, port, proxy, authentication, null, 0);
```
Comment (rsandell) [inline src/main/java/com/sonymobile/tools/gerrit/gerritevents/ssh/SshConnectionFactory.java]: `GerritDefaultValues.DEFAULT_GERRIT_SSH_CONNECTION_TIMEOUT` I guess?

**Comment by Jakohbae** [comment]
Thank you for your remarks.
I changed the files according to your requests.

**Comment by bartcubbins** [comment]
@jakohbae better rebase your PR and squash some commits it will be much cleaner

**Comment by Jakohbae** [comment]
@bartcubbins thx

**Comment by rsandell** [comment]
From a code review perspective rebasing and squashing is not a good because then I can't see what has changed since I last reviewed and need to do a full review again.
Better to just add commits to the branch on top of what is already reviewed.

**Comment by Jakohbae** [comment]
@rsandell Thanks for your input. Merry Christmas. :)

**Pull request merged**